docs/kubernetes/user-guides/windows.md

```diff
@@ -0,0 +1,33 @@
+# Kubernetes CSI Windowos User Guide
+
+>**Attention:** CSI Windows Beta is only available in the driver version v1.1.0+
+>**Attention:** CSI Windows Alpha is only available in the driver version v1.0.0-v1.0.*
+
+### Install CSI Proxy binary
+CSI proxy can be installed as binary or run as a Windows service on each Windows node. Please see details on CSI Proxy [installation and usage page](https://github.com/kubernetes-csi/csi-proxy/blob/master/README.md#usage).
+
+If you are using kube-up to start a GCE Kubernetes cluster, starting Kubernetes 1.19, [CSI Proxy Alpha](https://github.com/kubernetes-csi/csi-proxy/releases/tag/v0.1.0) binary is automatically installed during node start up. In Kubernetes 1.20, [CSI Proxy Beta.1 v0.2.1](https://github.com/kubernetes-csi/csi-proxy/releases/tag/v0.2.1) is installed and running as a windows service to improve its stablibity. For GKE cluster, starting from 1.18, CSI proxy will be installed automatically.
```

Comment:
I would put GKE on a new paragraph to make it stand out more.

Reply:
updated.